api/src/main/java/org/openmrs/Allergy.java

```diff
@@ -158,16 +158,36 @@ public void setSeverity(Concept severity) {
 	
 	/**
 	 * @return Returns the comment
+	 * @deprecated use getComments() instead
```

Comment:
Did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Deprecation